package.json

```diff
@@ -334,7 +333,8 @@
       "esbuild",
       "postcss",
       "prettier-plugin-tailwindcss",
-      "styled-jsx"
+      "styled-jsx",
+      "pdfjs-dist"
```

Comment:
I'm curious why that's needed, given we don't have it in our dependencies anymore?

Reply:
depcheck complains otherwise that we use it in next.config.js (to manually copy the pdf.worker.min.js and cMaps from it)